shared-bindings/wifi/Radio.c

```diff
@@ -283,7 +283,7 @@ const mp_obj_property_t wifi_radio_ap_info_obj = {
                (mp_obj_t)&mp_const_none_obj },
 };
 
-//|     def ping(self, ip, *, timeout: float = 0.5) -> float:
+//|     def ping(self, ip: wifi.Radio, *, timeout: float = 0.5) -> float:
```

Comment:
I think this should be an `ipaddress.IPv4Address`.

Reply:
Good catch! Fixed up now